android/src/main/java/io/sentry/RNSentryModule.java

```diff
@@ -92,12 +92,18 @@ public void startWithOptions(final ReadableMap rnOptions, Promise promise) {
             if (rnOptions.hasKey("dist") && rnOptions.getString("dist") != null) {
                 options.setDist(rnOptions.getString("dist"));
             }
-            if (rnOptions.hasKey("enableAutoSessionTracking") && rnOptions.getBoolean("enableAutoSessionTracking")) {
-                options.setEnableSessionTracking(true);
+            if (rnOptions.hasKey("enableAutoSessionTracking")) {
+                options.setEnableSessionTracking(rnOptions.getBoolean("enableAutoSessionTracking"));
             }
             if (rnOptions.hasKey("sessionTrackingIntervalMillis")) {
                 options.setSessionTrackingIntervalMillis(rnOptions.getInt("sessionTrackingIntervalMillis"));
             }
+            if (rnOptions.hasKey("enableNdkScopeSync")) {
+                options.setEnableScopeSync(rnOptions.getBoolean("enableNdkScopeSync"));
+            }
+            if (rnOptions.hasKey("attachStacktrace")) {
```

Comment:
do we already offer `attachThreads`? I'd suggest adding it as well.
it's now disabled by defaut

Reply:
We actually do, but it's right below that line and not an if statement like everything else. Let me refactor that.